app/src/views/ResultPage.vue

```diff
@@ -74,15 +74,15 @@
                 <td>Einsatzort</td>
                 <td v-html="selectedPost.location"></td>
               </tr>
-              <tr class="pt-1" v-if="selectedPost.title">
+              <tr class="pt-1 hide-copyright" v-if="selectedPost.title">
```

Comment:
Do you think a hide-copyright class is really necessary?

Reply:
Its convenient considering you need to apply this to atleast two different parts of the DOM and hiding the copyright requires conditional syntax for the shameOnYouIfYouRemove attribute. What would you do differently?

Reply:
I would have just added a css rule

```
.shameOnYouIfYouRemove { 
  display: none;
}
```

Otherwise I would have copied the rule from the source

```
.copyright, .copy, .cpy, strong[class^="copyright"], strong[class^="cpy"], strong[class^="copy"] {
    clear: both;
    padding: 10px 0px;
    display: none;
}
https://www.ehrenamtssuche-hessen.de/resources/css/styles.css?t=1594206724282
```

This would guarantee that the tag is hidden everywhere.